### PR TITLE
V1.4.0 ssr-2118 MobileTimePicker

### PR DESCRIPTION
**Pull request by h000780**
## Checklist

- [ ] Read CONTRIBUTING.md([for v1](https://github.com/kintone-labs/kintone-ui-component/master/CONTRIBUTING.md), [for v0](https://github.com/kintone-labs/kintone-ui-component/v0_dev/CONTRIBUTING.md))
- [ ] Updated documentation if it is required.
- [ ] Added tests if it is required.
- [ ] Passed lint and test command.

### CODE REVIEW

**Review by trung-doan — APPROVED** [review]

File: src/mobile/time-picker/test/value.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      expect(selectHourEl.value).to.be.equal("13");
+      expect(selectMinuteEl.value).to.be.equal("15");
+    });
+
+    it('should be "13:15" when changed to "13:15" by setter', async () => {
+      const container = new MobileTimePicker({ value: "11:15" });
+      const el = await fixture(container);
+      container.setAttribute("value", "13:15");
+      await elementUpdated(container);
+      el.setAttribute("value", "13:15");
```
Comment (dong0) [inline src/mobile/time-picker/test/value.test.ts:39]: This line can be deleted.
```suggestion
```

File: src/base/datetime/mobile-time/index.ts [hunk 2]
```diff
@@ -26,11 +29,13 @@ type Time = {
   suffix?: string;
 };
 
+// eslint-disable-next-line kuc-v1/no-using-generate-guid-function
 export class BaseMobileTime extends KucBase {
+  @property({ type: String }) GUID = "";
```
Comment (dong0) [inline src/base/datetime/mobile-time/index.ts:34]: GUID should be changed to lower case.

File: src/mobile/time-picker/test/value.test.ts [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      ) as HTMLSelectElement;
+      const selectMinuteEl = el.querySelector(
+        ".kuc-base-mobile-time__group__minutes"
+      ) as HTMLSelectElement;
+
+      expect(selectHourEl.value).to.be.equal("13");
+      expect(selectMinuteEl.value).to.be.equal("15");
+    });
+
+    it('should be "13:15" when changed to "13:15" in constructor', async () => {
```
Comment (dong0) [inline src/mobile/time-picker/test/value.test.ts:34]:
```suggestion
    it('should be "13:15" when changed to "13:15" by setter', async () => {
```

File: src/mobile/time-picker/index.ts [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+          font-size: 13px;
+          display: inline-block;
+          vertical-align: top;
+          width: 100%;
+        }
+        kuc-mobile-time-picker[hidden] {
+          display: none;
+        }
+        .kuc-mobile-time-picker__group {
+        }
```
Comment (dong0) [inline src/mobile/time-picker/index.ts:164]: No CSS code here, these lines can be removed.
```suggestion
```

File: src/mobile/time-picker/index.ts [hunk 5]
```diff
@@ -53,12 +75,34 @@ export class MobileTimePicker extends KucBase {
...
+    this.value = timeValueConverter(this.value);
+  }
+
   update(changedProperties: PropertyValues) {
-    if (changedProperties.has("value") && this.value) {
-      if (!validateTimeValue(this.value)) {
-        throw new Error(FORMAT_IS_NOT_VALID);
+    if (changedProperties.has("value") && !this._isSelectError) {
+      if (this.value === undefined) {
+        this._inputValue = "";
```
Comment (dong0) [inline src/mobile/time-picker/index.ts:101]: When INVALID_TIME_FORMAT error is displayed on UI, after set value undefined, the UI is empty, but error is still displayed.

**Pull request merged**